src/django_components/app_settings.py

```diff
@@ -115,5 +118,9 @@ def _validate_context_behavior(self, raw_value: ContextBehavior) -> ContextBehav
             valid_values = [behavior.value for behavior in ContextBehavior]
             raise ValueError(f"Invalid context behavior: {raw_value}. Valid options are {valid_values}")
 
+    @property
+    def TAG_FORMATTER(self) -> Union["TagFormatterABC", str]:
```

Comment:
Updated as per feedback, the settings now allow to set only a single TagFormatter, which is then used for component tags.